api/client.go

```diff
@@ -720,6 +720,42 @@ func (c *Client) SetMaxRetries(retries int) {
 	c.config.MaxRetries = retries
 }
 
+func (c *Client) SetDisableKeepAlives(disable bool) {
```

Comment:
I don't think these are used anymore.

Reply:
They aren't, I've opted to keep them because they might be useful some day, but I could remove if needed.

Reply:
Removed them anyway.